main.py

```diff
@@ -2,11 +2,19 @@
 from Profiles.ProfilesAPI import profiles_api
 from flask import Flask
 from db import db
+from db import *
 
 
 # Write your flask code here
 
 app = Flask(__name__)
 
+app.config['SECRET_KEY'] = AUTH_SECRET_KEY
 app.register_blueprint(profiles_api, url_prefix="/profiles")
 app.register_blueprint(auth_api, url_prefix="/auth")
+
+@app.route("/")
+def hello_world():
+    return "Welcome to the database!"
+
+app.run
```

Comment:
This does not do anything. Please look into the correct way to start the app on the notes. app.run is correct but needs more arguments